.infrastructure/50_storage.tf

```diff
@@ -87,6 +87,14 @@ module "dynamodb_webinar_questions" {
       name = "createdAt"
       type = "S"
     },
+    {
+      name = "hiddenBy"
+      type = "S"
+    },
+    {
+      name = "highlightedBy"
+      type = "S"
+    },
```

Comment:
Attributes blocks [defines the key schema for the table and indexes](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_AttributeDefinition.html), this means that all defined attributes must be indexed, otherwise they should not be defined or terraform will [raise an error](https://github.com/hashicorp/terraform-provider-aws/blob/v5.36.0/internal/service/dynamodb/table.go#L2381-L2383).

```suggestion
```